### ISSUE TITLE
SyntaxError: invalid syntax

### ISSUE THREAD

**Issue by kmkale** [rpgreen/apilogs#6]
Hi,

I installed apilogs today with pip install apilogs. The install was successful. But when I invoke the command, I get below error:

apilogs groups --profile TestMgmtAdmin
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.4/bin/apilogs", line 7, in <module>
    from apilogs.bin import main
  File "/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/site-packages/apilogs/__init__.py", line 2, in <module>
    from .core import AWSLogs  # noqa
  File "/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/site-packages/apilogs/core.py", line 168
    print "Error fetching logs for Lambda function {0} with group {1}. This function may need to be invoked.".format(fxn, lambda_group, e)
                                                                                                            ^
SyntaxError: invalid syntax

I am getting the same error for any of the accounts I have set up profiles for. i.e. dev, test and prod. 

PS: The awslogs command installs and runs successfully.

**Comment by omatrot** [comment]
Same issue here. With the following command:

`apilogs get --api-id myid --stage prod --watch`

```
Traceback (most recent call last):
  File "/usr/local/bin/apilogs", line 7, in <module>
    from apilogs.bin import main
  File "/usr/local/lib/python3.6/site-packages/apilogs/__init__.py", line 2, in <module>
    from .core import AWSLogs  # noqa
  File "/usr/local/lib/python3.6/site-packages/apilogs/core.py", line 168
    print "Error fetching logs for Lambda function {0} with group {1}. This function may need to be invoked.".format(fxn, lambda_group, e)
                                                                                                            ^
SyntaxError: invalid syntax
```

I'm afraid this project is not maintained.

**Comment by rpgreen** [comment]
This is a different command than the previous report. Does the log group
exist?

**Comment by omatrot** [comment]
No command is actually working for me. Same error.

**Comment by rpgreen** [comment]
@kmkale Does it work if you use 'apilogs get ...'?
@omatrot can you confirm if the log groups actually exist? Have you invoked the API and is logging turned on?

**Comment by kmkale** [comment]
@rpgreen  same error for get too.

`apilogs get --api-id aaskjd765 --stage dev --watch --profile DevAdmin`

```
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.4/bin/apilogs", line 7, in <module>
    from apilogs.bin import main
  File "/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/site-packages/apilogs/__init__.py", line 2, in <module>
    from .core import AWSLogs  # noqa
  File "/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/site-packages/apilogs/core.py", line 168
    print "Error fetching logs for Lambda function {0} with group {1}. This function may need to be invoked.".format(fxn, lambda_group, e)
                                                                                                            ^
SyntaxError: invalid syntax
```
The api has been called, logs exist.

**Comment by rpgreen** [comment]
Thanks. This appears to be related to python 3. PRs are appreciated. It should be pretty straightforward as per https://docs.python.org/3/howto/pyporting.html

Alternatively, you can run with Python 2.7

**Comment by jonathan-kosgei** [comment]
@rpgreen I have the same issue with python 2.7 and python 3.6

```
jonathan@ubuntu ~> python --version
Python 2.7.14
jonathan@ubuntu ~> apilogs groups
Traceback (most recent call last):
  File "/usr/local/bin/apilogs", line 7, in <module>
    from apilogs.bin import main
  File "/usr/local/lib/python3.6/dist-packages/apilogs/__init__.py", line 2, in <module>
    from .core import AWSLogs  # noqa
  File "/usr/local/lib/python3.6/dist-packages/apilogs/core.py", line 168
    print "Error fetching logs for Lambda function {0} with group {1}. This function may need to be invoked.".format(fxn, lambda_group, e)
                                                                                                            ^
SyntaxError: invalid syntax
```

```
jonathan@ubuntu ~> python --version
Python 3.6.3
jonathan@ubuntu ~> apilogs groups
Traceback (most recent call last):
  File "/usr/local/bin/apilogs", line 7, in <module>
    from apilogs.bin import main
  File "/usr/local/lib/python3.6/dist-packages/apilogs/__init__.py", line 2, in <module>
    from .core import AWSLogs  # noqa
  File "/usr/local/lib/python3.6/dist-packages/apilogs/core.py", line 168
    print "Error fetching logs for Lambda function {0} with group {1}. This function may need to be invoked.".format(fxn, lambda_group, e)
                                                                                                            ^
SyntaxError: invalid syntax

```

**Comment by rpgreen** [comment]
Thanks for reporting. Looks like this was just an issue in the error handling code that was already fixed in mainline. I've pushed the change to pypi as version 1.2. Upgrade with 'pip install apilogs --upgrade' and you should see a proper error message. Let me know if you're still having issues

**Comment by jonathan-kosgei** [comment]
Thanks for the quick response. I'll test this first thing in the morning. Thanks again.

**Comment by kmkale** [comment]
After the update the command does not throw the error but does not return any logs either. 
Tried this command:
apilogs get --api-id myAPIiD --stage dev --start='23/2/2018 20:00' --end='23/2/2018 22:00' --profile MyProfileName

I know the logs are present for the date range I am asking for. I can see the in cloudwatch. I have tried to supply the date in different formats. No output. Is there a verbose or debug mode where I can see what the command is doing?
